var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -249,3 +251,31 @@ def install(self, spec, prefix):
                 filter_file('mpcc_r)', 'mpcc_r mpifcc)', f, string=True)
                 filter_file('mpc++_r)', 'mpc++_r mpiFCC)', f, string=True)
                 filter_file('mpifc)', 'mpifc mpifrt)', f, string=True)
+
+    def _test_check_versions(self):
+        """Perform version checks on installed package binaries."""
+        spec_vers_str = 'version {0}'.format(self.spec.version)
+
+        checks = {
+            'ccmake': ([spec_vers_str], None),
+            'cmake':  ([spec_vers_str], None),
+            'cpack':  ([spec_vers_str], None),
+            'ctest':  ([spec_vers_str], None),
+        }
+
+        for exe in checks:
+            expected, status = checks[exe]
+            reason = 'test version of {0} is {1}'.format(exe, expected[0])
+            self.run_test(exe, ['--version'], expected, status, installed=True,
+                          purpose=reason, skip_missing=True)
```

Comment:
```suggestion
        for exe in ('cmake', 'ccmake', 'cpack', 'ctest'):
            reason = 'test version of {0} is {1}'.format(exe, spec_vers_str)
            self.run_test(exe, ['--version'], expected=[spec_vers_str], installed=True,
                          purpose=reason)
```

Reply:
As long as the status is `None|0` and the version is always the same, we don't need to list those for each executable.

Reply:
Fair enough.  Changed.